Networking/Networking/Model/Media/Media.swift

```diff
@@ -47,7 +47,7 @@ extension Media: Decodable {
         let date = try container.decodeIfPresent(Date.self, forKey: .date) ?? Date()
         let fileExtension = try container.decodeIfPresent(String.self, forKey: .fileExtension) ?? ""
         let mimeType = try container.decodeIfPresent(String.self, forKey: .mimeType) ?? ""
-        let src = try container.decodeIfPresent(URL.self, forKey: .src)?.absoluteString ?? ""
+        let src = try container.decodeIfPresent(String.self, forKey: .src) ?? ""
```

Comment:
Perhaps add a comment here that we cannot decode URL directly since the image src URL may contain non Latin characters that have not been percent encoded.